syft/workers/base.py

```diff
@@ -182,8 +182,6 @@ def __init__(
 
         # storage object for crypto primitives
         self.crypto_store = PrimitiveStorage(owner=self)
-        # declare the plans used for crypto computations
-        sy.frameworks.torch.mpc.fss.initialize_crypto_plans(self)
```

Comment:
I don't think you really meant to remove that

Reply:
I really did, because it makes the tests fail and shouldn't have been there in the first place.

Reply:
I opened an issue for it, but there hasn't been a fix: https://github.com/OpenMined/PySyft/issues/3464

Reply:
I can move the fix to a separate PR, though. 🤷

Reply:
Removed and submitted as #3525